nidirect_driving_instructors/nidirect_driving_instructors.module

```diff
@@ -38,29 +39,56 @@ function nidirect_driving_instructors_form_alter(&$form, &$form_state, &$form_id
     $form['sort_by']['#type'] = 'hidden';
     $form['sort_order']['#type'] = 'hidden';
 
-    // Add additional form classes.
-    $form['#attributes']['class'][] = 'search-form';
-    $form['#attributes']['class'][] = 'search-form--driving-instructors';
+    // Add some form help.
+    $help = [
+      '#theme' => 'item_list',
+      '#list_type' => 'ul',
+      '#items' => [
+        t('Enter your postcode to find instructors who work in your area'),
+        t('You can also find instructors by name or ADI/AMI number if you know it'),
+      ],
+    ];
+    $form['help'] = [
+      '#type' => 'item',
+      'content' => $help,
+    ];
 
     // Add postcode placeholder example.
     $form['search_api_fulltext']['#placeholder'] = t("e.g. 'BT1'");
 
-    // Update exposed Categories element and text.
-    $form['categories']['#type'] = 'radios';
-    $form['categories']['#options'] = [];
-    $form['categories']['#options']['All'] = t('All');
-    $form['categories']['#options']['1160'] = t('Driving instructors');
-    $form['categories']['#options']['1161'] = t('Motorcycle instructors');
+    // Hide exposed categories radios.
+    $form['categories']['#type'] = 'hidden';
+
+    // Instead, output radios for choosing facet filters.
+    // See driving instructors facet source configuration (/admin/config/search/facets)
+    // for the correct filter key to use.
+    $form['f[0]'] = [
+      '#id' => Html::getUniqueId('edit-categories'),
+      '#type' => 'radios',
+      '#title' => t('Filter by type of tuition'),
+      '#title_display' => 'above',
+      '#options' => [
+        'categories:all' => t('All'),
+        'categories:1160'  => t('Driving instructors'),
```

Comment:
There's a potential brittle area here with the term ids baked into the form structure values. I figure these are unlikely to change much as much of the IA is established, but ideally, these term ids would come from using `\Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadByProperties(['name' => 'Driving instructors']);` .... then extracting the term ids from the object that's returned.

Reply:
Thanks Johan - could you take another look at this please?